doc/.readthedocs.yaml

```diff
@@ -9,7 +9,7 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    golang: "1.21"
+    golang: "1.22.0"
```

Comment:
Yeah, RTD does not yet support Go 1.22 and they certainly don't support expressing the version with 3 digits.
I asked for Go 1.22 in https://github.com/readthedocs/readthedocs.org/issues/11189